content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Operator Quick Start Evaluation Install
```

Comment:
For users that don't know what an Operator is, this title can be confusing. `Quick Start Evaluation Install with Operator` maybe? Or if this is going to be the only Quick Start Evaluation method, then saying Operator might not be needed.

Reply:
done.